src/lib.rs

```diff
@@ -252,10 +252,23 @@ pub enum CursorIcon {
     /// Indicates that something can be zoomed in. Often rendered as a
     /// magnifying glass with a "-" in the center of the glass.
     ZoomOut,
+
+    /// Indicates that the user will select the action that will be carried out.
+    ///
+    /// This is a non-standard extension of the w3c, that will only be emitted
+    /// when used with a wayland backend.
```

Comment:
```suggestion
    /// This is a non-standard extension of the w3c, that will only be emitted
    /// when used with a wayland backend.
```

There's no `wayland backend`. We should just name that it's a non-standard thing and probably that's about it.

Reply:
took the mention of the wayland backend from https://github.com/rust-windowing/cursor-icon/pull/17#discussion_r2103191586, but yeah on second though it does sound a little weird.

for consistency with the docs for the non-standard keyboard types (see [keyboard-types/src/code.rs](https://github.com/rust-windowing/keyboard-types/blob/5135e103e9566a3a0283ab4b2612b534a247d71a/src/code.rs#L423-L462)), would it maybe make sense to say "This is a non-standard extension of the w3c supported by wayland"?

Reply:
did it like that now, i'm open to changing it though